covid19datavis.md

```diff
@@ -0,0 +1,32 @@
+# COVID 19 Data Exploration and Visualization
+
+**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+
+### :pushpin: Step 1
+**TITLE:**    
+{{COVID 19 Data Exploration and Visualization}}
+
+**TOPIC:**    
+{{Probability and Statistics}}
+
+**DESCRIPTION (5-7+ sentences):**    
+{{Plot of Incident cases Vs. Date (automatically updated with data source)
+Plot of R number Vs. Date 
+Plot in Tableau (optional)
+Forecast of number of cases in the future (optional)}}
+
```

Comment:
Your description should be an actual paragraph, not just a set of bullet points. It's basically writing down what you would say if you tried to explain the concept of your blog in person to someone else.